packages/salesforcedx-vscode-replay-debugger/src/constants.ts

```diff
@@ -18,4 +18,6 @@ export const EVENT_METHOD_EXIT = 'METHOD_EXIT';
 export const EVENT_STATEMENT_EXECUTE = 'STATEMENT_EXECUTE';
 export const EXEC_ANON_SIGNATURE = 'execute_anonymous_apex';
 export const GET_LINE_BREAKPOINT_INFO_EVENT = 'getLineBreakpointInfo';
-export const LINE_BREAKPOINT_INFO_REQUEST = 'lineBreakpointInfo';
\ No newline at end of file
+export const LINE_BREAKPOINT_INFO_REQUEST = 'lineBreakpointInfo';
+export const VF_APEX_CALL_START = 'VF_APEX_CALL_START';
```

Comment:
Group this with the other events above and use the same naming convention.

Reply:
Renamed them to EVENT_VF_APEX_CALL_START and EVENT_VF_APEX_CALL_END and grouped them with the other events. I'll update this review shortly with the changes.